client/my-sites/exporter/exporter.jsx

```diff
@@ -43,6 +43,7 @@ export default React.createClass( {
 		const siteId = this.props.site.ID;
```

Comment:
on the above declarations, why do we have two lines? I know this isn't in the current PR, but I found my way up here when looking at line 46. Do you think it would be worth merging into one statement?

```js
const {
	exportStatusFetch,
	isExporting,
	postType,
	setPostTypes,
	shouldShowProgress,
	startExport,
} = this.props;
```

This style suits me well because it fits closer into our coding standard for making diffs clearer, plus it's alphabetical, which is visually more appealing me at least.